packages/ui/src/molecules/RadioGroup/stories/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react'
+import type { Story } from '@storybook/react'
+
+import type { RadioGroupProps, RadioOptionProps } from '..'
+import { RadioGroup, RadioOption } from '..'
+import mdx from './RadioGroup.mdx'
+
+export const RadioGroupSimple: Story<RadioGroupProps & RadioOptionProps> = ({
+  onChange,
+  ...args
+}) => {
+  const [option, setOption] = useState<string | number>('')
+
+  return (
+    <>
+      <RadioGroup
+        {...args}
+        name="radio-group"
+        onChange={(v) => {
+          setOption(v.currentTarget.value)
+          onChange?.(v)
+        }}
+        selectedValue={option}
+      >
+        <RadioOption value="radio-1" label="Radio 1">
+          <span>Radio 1</span>
+        </RadioOption>
+        <RadioOption value="radio-2" label="Radio 2">
+          <span>Radio 2</span>
+        </RadioOption>
+      </RadioGroup>
+    </>
+  )
+}
+
+export const RadioOptionWithChildren: Story<
+  RadioGroupProps & RadioOptionProps
+> = ({ onChange, ...args }) => {
+  const [option, setOption] = useState<string | number>('')
+
+  return (
+    <>
+      <RadioGroup
+        {...args}
+        name="radio-group"
```

Comment:
The storybook is glitching a little bit because all RadioGroups have the same name. Would you change it pls?

Reply:
On the docs page!